CMakeLists.txt

```diff
@@ -131,7 +131,7 @@ if(${VELOX_BUILD_BENCHMARKS})
   set(VELOX_BUILD_TEST_UTILS ON)
   set(VELOX_BUILD_TESTING OFF)
   set(VELOX_ENABLE_EXAMPLES OFF)
-  set(VELOX_ENABLE_S3 OFF)
+  set(VELOX_ENABLE_S3 ON)
```

Comment:
afaik this is only needed for io tuning right? As s3 is quite the big dependency it might be good to add an additional option? Though it follow what ever @kgpai thinks is best in this case.